assets/scss/theme-colours.scss

```diff
@@ -0,0 +1,28 @@
+@import 'colours';
+
+.page-colour--teal {
```

Comment:
I like this solution but thinking about scalability im thinking.

1. We split each theme colour into its own sass file to prevent theme-colours.scss getting too big
OR
2. Or to prevent repetition we create a set of colour variables (we can always add new ones) that get passed into a SASS function so we dont have the sass repeated per theme colour

Reply:
Yes, I agree.  When we get to do the colours I guess we can decide then.  This is just an exemplar in case we need to show the principle.

Reply:
Is that part of this ticket, or the next ticket Malcolm is going to pick up?

Reply:
Next ticket.  But I did this little exemplar here just in case anyone wanted a demonstration of how it might work.